back/appsync/schema.graphql

```diff
@@ -123,6 +123,9 @@ type Mutation {
     snoozeEmails(userId:String!, token:String, days:Int): Result!
     @aws_api_key @aws_cognito_user_pools
 
+    unsubscribe(userId: String!, token: String): UnsubscribeResult!
+    @aws_api_key @aws_cognito_user_pools
```

Comment:
ahy api key auth? its for not authed users and unsubscribe supposed be for authed?